### PR TITLE
Onboard request ctas

### LINKED ISSUE #976: WEB / HUBS - add claim request to No SOL modal on create actions

### PR DESCRIPTION
**Pull request by ejf89**
closes #976 #978

### CODE REVIEW

File: js/sdk/src/components/EmailCapture.js [hunk 1]
```diff
@@ -57,51 +52,91 @@ const EmailCaptureSchema = Yup.object().shape(
...
-
-  useEffect(() => {
-    if (open) {
-      logEvent('email_request_initiated', 'engagement')
-    }
-  }, [open])
+  const [pending, setPending] = useState(false)
+  const [userVerifications, setUserVerifications] = useState(undefined)
+  const [soundcloudAccount, setSoundcloudAccount] = useState(undefined)
+  const [twitterAccount, settwitterAccount] = useState(undefined)
```
Comment (bbnickell) [inline js/sdk/src/components/EmailCapture.js:71]: Very minor small nitpick thing here, but we should rename this to setTwitterAccount to maintain consistent camelcase.

File: js/sdk/src/components/EmailCapture.js [hunk 2]
```diff
@@ -57,51 +52,91 @@ const EmailCaptureSchema = Yup.object().shape(
...
       setSubmitButtonText('Submit')
     }
   }, [formValues])
 
+  useEffect(() => {
+    if (userVerifications) {
+      setSoundcloudAccount(
+        getVerificationValue(userVerifications, 'soundcloud')
+      )
+      settwitterAccount(getVerificationValue(userVerifications, 'twitter'))
```
Comment (bbnickell) [inline js/sdk/src/components/EmailCapture.js:105]: camel case consistency here as well, setTwitterAccount

File: js/sdk/src/components/EmailCapture.js [hunk 3]
```diff
@@ -57,51 +52,91 @@ const EmailCaptureSchema = Yup.object().shape(
...
+          setShowSuccessInfo(true)
+          enqueueSnackbar('Email request submitted', { variant: 'success' })
+        }
         logEvent('email_request_success', 'engagement', {
           email: formValues.email,
         })
-        setShowSuccessInfo(true)
       } catch (error) {
         console.warn('email form error', error)
         logEvent('email_request_success', 'engagement', {
```
Comment (bbnickell) [inline js/sdk/src/components/EmailCapture.js:135]: should this be email_request_error or something to that effect since this occurring in the catch block?

**Review by bbnickell — APPROVED** [review]
Mostly looks good to me as far as I can tell, just had a couple very small comments.

**Comment by ikjudd** [comment]
left a comment on the vercel build, otherwise +1 what Brandon said / looks good

**Review by facing-n — APPROVED** [review]

**Comment by facing-n** [comment]
@ejf89 Approved - but i think we should fix the copy on the confirm step
Screenshot 2023-05-02 at 12 36 28 PM

should say something more relevant to the action: "Thanks for requesting some SOL to get started publishing on Nina" or this is also a place to introduce Nina Artists Program

**Review by bbnickell — APPROVED** [review]
PR looks good 👍 Approved.

**Pull request merged**